### DISCUSSION TITLE
Neovim vs Helix: Which is the best Vi/Vim style modal editor?

### DISCUSSION THREAD

**Discussion by David-Else** [helix-editor/helix#2183] [General]
https://elsewebdevelopment.com/neovim-vs-helix-which-is-the-best-vi-vim-style-modal-editor/

I hope people like it! :)

**Comment by juniway** [comment]
Does helix have **session management** and **tree-view file explorer**?

**Reply by David-Else** [reply]
No, but Neovim and Vim do have basic session management.
